File: visual basic calc/calc.vb
Module calc

    Sub Main()
        Static res As Integer

        Console.ForegroundColor = ConsoleColor.Green
        Console.Title = "Calculator"

        Console.WriteLine("pls enter your first number")
        Static Num1 As String = Console.ReadLine()
        Console.WriteLine("pls enter your second number")
        Static Num2 As String = Console.ReadLine()
        Console.WriteLine("pls enter your opreation sign")
        Static Sign As String = Console.ReadLine()

        If Sign = "+" Then
            res = Convert.ToInt64(Num1) + Convert.ToInt64(Num2)
            Console.WriteLine(res)
        ElseIf Sign = "-" Then
            res = Convert.ToInt64(Num1) - Convert.ToInt64(Num2)
            Console.WriteLine(res)
        ElseIf Sign = "*" Then
            res = Convert.ToInt64(Num1) * Convert.ToInt64(Num2)
            Console.WriteLine(res)
        ElseIf Sign = "/" Then
            res = Convert.ToInt64(Num1) / Convert.ToInt64(Num2)
            Console.WriteLine(res)
        Else
            Console.WriteLine("this is not a vaild sign")
        End If
        Console.ReadKey()
    End Sub

End Module

File: c# calc/calc.cs
using System;

namespace csharp
{
    class calc
    {
        static void Main(string[] args)
        {
            float res;
            
            Console.WriteLine("pls enter your first number");
            string num1 = Console.ReadLine();
            Console.WriteLine("pls enter your second number");
            string num2 = Console.ReadLine();
            Console.WriteLine("pls enter your operation sign");
            string sign = Console.ReadLine();

            if (sign == "+")
            {
               res = Convert.ToInt32(num1) + Convert.ToInt32(num2);
               Console.WriteLine("your answer is "+res);
            }
            else if (sign == "-")
            {
                res = Convert.ToInt32(num1) - Convert.ToInt32(num2);
                Console.WriteLine("your answer is "+res);
            }
            else if (sign == "*")
            {
                res = Convert.ToInt32(num1) * Convert.ToInt32(num2);
                Console.WriteLine("your answer is "+res);
            }
            else if (sign == "/")
            {
                res = Convert.ToInt32(num1) / Convert.ToInt32(num2);
                Console.WriteLine("your answer is "+res);
            }

            Console.ReadKey();
        }
    }
}
//c# rating 10👍/10